scripts/_common.sh

```
@@ -1,20 +1,35 @@
+  local final_path=$1
+  local src_file_path="$YNH_CWD/../conf/app.src"
+  # In case of restore script the src file is in an other path.
+  # So try to use the restore path if the general path point to no file.
+  if [ ! -e "$src_file_path" ]; then
+      src_file_path="$YNH_CWD/../settings/conf/spp.src"
+  fi
 
-#=================================================
-# PERSONAL HELPERS
-#=================================================
+  local src_url=$(grep 'SOURCE_URL=' "$src_file_path" | cut --delimiter='=' --fields=2-)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)